WordOfTheDay.Repository/Entities/WordContext.cs

```diff
@@ -12,5 +12,15 @@ public WordContext (DbContextOptions<WordContext> options)
         public WordContext()
         {
         }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Word>()
+                .HasIndex(w => new { w.AddTime, w.Email })
```

Comment:
I suppose this index should be unique. User can not add more than one word a day

Reply:
done